### ISSUE TITLE
Missing Test Case - 322. Coin Change

### ISSUE THREAD

**Issue by jpbattaille** [LeetCode-Feedback/LeetCode-Feedback#7456]
#### Your LeetCode username

jpbattaille

#### Category of the bug
- [ ] Question
- [ ] Solution
- [ ] Language
- [ x] Missing Test Cases 

#### Description of the bug

allowed 10000 coins, my program would fail for 10000 coins exactly

#### Code you used for Submit/Run operation

```
class Solution:
    def coinChange(self, coins: List[int], amount: int) -> int:
       
        dp = [10000 for i in range(amount+1)]
        dp[0] = 0

        for i in range(amount+1):
            for coin in coins:
                if i<coin: continue
                dp[i] = min(dp[i-coin]+1, dp[i])


        if dp[amount] ==  10000: return -1
        return dp[amount]
```

#### Language used for code

Python3

#### Expected behavior

the correct answer is 10000 and not -1

#### Additional context

basically we need to add the border case:
expected 10000
amount 10000
coins [1]

**Comment by LC-Pam** [comment]
Thank you for reaching out to us. To help us further assess this issue and reproduce this issue on our end, would you please share the following information with us:
The specific test case that you use.
We appreciate your time and support. If you have any other questions, please don't hesitate to let us know.

**Comment by jpbattaille** [comment]
The test case was shared :) 
input : (amount =10000, coins= [1])
output = 10000
My code would give -1

**Comment by LC-Pam** [comment]
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by LC-Pam** [comment]
Thank you for your time. We've used your feedback to update the problem. Your LeetCode account has received 100 LeetCoins as the reward for this feedback. We appreciate your support!
